server/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
can change line 8 to have "jest --watchAll", this is nitpicking but obviously if we were to test you would have done that.